docs/guide/save.rst

```diff
@@ -30,3 +30,28 @@ For example, to save an image with low quality:
 
     with open('low_quality.jpg', 'wb') as f:
         i.save_as_jpeg(f, quality=40)
+
+Progressive JPEGs
+-----------------
+
+:meth:`~Image.save_as_jpeg` takes a ``progressive`` keyword argument, that can
+be used to make it output a progressive JPEG file:
+
+.. code-block:: python
+
+    with open('progressive.jpg', 'wb') as f:
+        i.save_as_jpeg(f, progressive=True)
+
+Optimized images
+----------------
+
+:meth:`~Image.save_as_jpeg` and :meth:`~Image.save_as_png` both take an
+``optimise`` keyword that when set to true, will output an optimized image.
+
+.. code-block:: python
+
+    with open('optimized.jpg', 'wb') as f:
+        i.save_as_jpeg(f, optimize=True)
```

Comment:
is it 'optimise' or 'optimize'? You have 'ise' in the description and 'ize' in the example.